src/components/DeviceInfo/DesktopInfo.js

```diff
@@ -27,4 +28,17 @@ const DesktopInfo = (props) => {
   )
 }
 
+DesktopInfo.defaultProps = {
+  detail: {
+    brand: 'Chưa có thông tin',
+    configuration: 'Chưa có thông tin',
+    size: 'Chưa có thông tin',
+    images: []
+  },
+}
+
+DesktopInfo.propTypes = {
+  detail: PropTypes.object.isRequired
```

Comment:
What if I pass the wrong type of `brand`, `configuration`, `size`, `images`?

Reply:
Noted, I will fix this by adding PropTypes.shape